HelpSource/Classes/Object.schelp

```diff
@@ -652,6 +694,55 @@ foo {
 }
 ::
 
+
+subsection::Handling 'non-understood' messages and writing 'wrapper' classes
+
+When an object receives a message that it does not understand one of the following two methods are called before an error is thrown: link::#-doesNotUnderstand:: and link::#-doesNotUnderstandWithKeys::.
+This means it is possible to redirect the message, calling any other code desired.
+
+link::#-doesNotUnderstand:: is quasi-deprecated, meaning it is unnecessary to implement but maintained for compatibility reasons.
```

Comment:
To me, "deprecate" is too strong. Perhaps we should just describe it as a choice. Maybe I feel emotionally attached to the message `doesNotUnderstand`!